avro_schemas/sunav2/sunav2.avsc

```diff
@@ -0,0 +1,363 @@
+        {
+            "name": "header_manufacturer",
+            "type": [
+                "null",
+                "string"
+            ],
+            "default": null,
+            "doc": "The very first part of the header, which should always be the four character 'SATS'"
+        },
+        {
+            "name": "header_serial_number",
+            "type": [
+                "null",
+                "string"
+            ],
+            "default": null,
+            "doc": "The manufacturer serial number of the SUNA v2 device"
+        },
+        {
+            "name": "header_light_frame",
+            "type": [
+                "null",
+                "boolean"
+            ],
+            "default": null,
+            "doc": "This field is true when the header string contains 'LF' (Light Frame) in between the manufacturer and the serial number indicating that the  data frame was captured with the lamp on. This field is false when 'DF' (Dark Frame) is in the header indicating that the lamp was off during the data frame."
+        },
+        {
+            "name": "year_and_day",
+            "type": [
+                "null",
+                "string"
+            ],
+            "default": null,
+            "doc": "Date, year and day of year"
+        },
+        {
+            "name": "time",
+            "type": [
+                "null",
+                "float"
+            ],
+            "default": null,
+            "doc": "Hours of day"
+        },
+        {
+            "name": "nitrate_concentration",
+            "type": [
+                "null",
+                "float"
+            ],
+            "default": null,
+            "doc": "The measured nitrate concentration in micromoles",
+            "__neon_units": "micromole"
+        },
+        {
+            "name": "nitrogen_in_nitrate",
+            "type": [
+                "null",
+                "float"
+            ],
+            "default": null,
+            "doc": "The measured amount of nitrogen in micrograms per liter",
+            "__neon_units": "microgramsPerLiter"
+        },
+        {
+            "name": "absorbance_254nm",
+            "type": [
+                "null",
+                "float"
+            ],
+            "default": null,
+            "doc": "Absorbance at 254 nanometers"
+        },
+        {
+            "name": "absorbance_350nm",
+            "type": [
+                "null",
+                "float"
+            ],
+            "default": null,
+            "doc": "Absorbance at 350 nanometers"
+        },
+        {
+            "name": "bromide_trace",
+            "type": [
+                "null",
+                "float"
+            ],
+            "default": null,
+            "doc": "Bromide trace in milligrams per liter",
+            "__neon_units": "milligramsPerLiter"
+        },
+        {
+            "name": "spectrum_average",
+            "type": [
+                "null",
+                "int"
```

Comment:
From the description, this doesn't look like an integer output. Has this schema been tested?